xtext-gradle-plugin/src/integTest/java/org/xtext/gradle/test/GradleBuildTester.xtend

```diff
@@ -28,6 +29,7 @@ class GradleBuildTester extends ExternalResource {
 			owner = this
 		]
 		gradle = GradleRunner.create.withProjectDir(rootProject.projectDir)
+		(gradle as DefaultGradleRunner).withJvmArguments('-Xmx512m', '-XX:MaxPermSize=512m')
```

Comment:
As discussed via mail, this indicates a memory leak that needs fixing. Xtend does not need this much permGenSpace